src/Components/OATGraphViewer/Internal/Utils.tsx

```diff
@@ -53,7 +53,9 @@ export const addTargetedRelationship = (
         '',
         sourceId,
         OAT_RELATIONSHIP_HANDLE_NAME,
-        relationship['target'],
+        Array.isArray(relationship['target'])
```

Comment:
is there a reason why we dont use `relationship.target` as below? Is it due to type casting might be needed if we use?
```suggestion
        Array.isArray(relationship.target)
```

Reply:
I think this is just legacy formatting from when the vendors wrote it. I can update it.

Reply:
done.